src/Internal.AspNetCore.SiteExtension.Sdk/Internal.AspNetCore.SiteExtension.Sdk.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Internal.AspNetCore.SiteExtension.Sdk</PackageId>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
+    <Serviceable>false</Serviceable>
+    <Description>Build targets and extensions to Microsoft.NET.Sdk. This package is intended for Microsoft use only.</Description>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <NuspecFile>$(MSBuildThisFileDirectory)$(MSBuildProjectName).nuspec</NuspecFile>
+    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSource>false</IncludeSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="$(Tooling_NewtonsoftJsonPackageVersion)" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="15.1.1012" />
```

Comment:
Are we sure we want these as hardcoded version numbers? @natemcmaster

Reply:
True. There are package versions available for this. Would be better to use those.

Reply:
fixed